Resources/Prototypes/DeltaV/GameRules/events.yml

```diff
@@ -132,13 +132,13 @@
     earliestStart: 25
   - type: MidRoundAntagRule
 
-#- type: entity
-#  parent: BaseMidRoundAntag
-#  id: RatKingSpawn
-#  components:
-#  if you uncomment this update this for antag refactor, below wont work
-#  - type: MidRoundAntagRule
-#    spawner: SpawnPointGhostRatKing
+- type: entity
+  parent: BaseMidRoundAntag
+  id: RatKingSpawn
+  components:
+  if you uncomment this update this for antag refactor, below wont work
```

Comment:
This shouldn't be here it was a whole comment so either comment it back or delete it